modules/modes/daycare.py

```diff
@@ -198,7 +198,8 @@ def pc_release():
                 yield from wait_until_task_is_active("Task_PokemonStorageSystem", "A")
             else:
                 yield from wait_until_task_is_active("Task_PCMainMenu", "A")
-            yield from wait_for_n_frames(8)
+            # Non English versions require at least 10 frames
```

Comment:
You're right, I remove the comment. But unlike the other one, I tested it and this frame change is needed atleast on french version, otherwise it doesn't release Pokémons